modules/library/main/src/main/java/org/geotools/filter/function/FilterFunction_arrayAnyMatch.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2019, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.filter.function;
+
+import static org.geotools.filter.capability.FunctionNameImpl.parameter;
+
+import java.util.Arrays;
+import org.geotools.filter.FunctionExpressionImpl;
+import org.geotools.filter.capability.FunctionNameImpl;
+import org.opengis.filter.capability.FunctionName;
+
+public class FilterFunction_arrayAnyMatch extends FunctionExpressionImpl {
+    public static FunctionName NAME =
+            new FunctionNameImpl(
+                    "arrayAnyMatch",
+                    parameter(
+                            "arrayAnyMatch",
```

Comment:
Feedback: The name "arrayAnyMatch" is still odd to me when the usual function is called `IN`, would "InArray" work for you? This would also reverse the order of the arguments:
```
InArray( candidate, array )
```

Reply:
I agree going to rename to inArray, and removing boilerplate